### PR TITLE
MKC: added Take The Bait

### PR DESCRIPTION
**Pull request by Morgenmvffel**
added MKMs Take the Bait

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/take_the_bait.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Take the Bait
+ManaCost:2 R W
+Types:Instant
+A:SP$ Effect | Cost$ 2 R W | ReplacementEffects$ RPrevent | OpponentTurn$ True | ActivationPhases$ BeginCombat->EndCombat | SubAbility$ DBUnTapAll | SpellDescription$ Prevent all combat damage that would be dealt to you and planeswalkers you control this turn. Untap all attacking creatures and goad them. After this phase, there is an additional combat phase.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/take_the_bait.txt:4]: no Cost needed

File: forge-gui/res/cardsfolder/upcoming/take_the_bait.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+SVar:RPrevent:Event$ DamageDone | Prevent$ True | ValidTarget$ You,Planeswalker.YouCtrl
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/take_the_bait.txt:5]: missing IsCombat$

Comment (Morgenmvffel) [reply forge-gui/res/cardsfolder/upcoming/take_the_bait.txt]: added

File: forge-gui/res/cardsfolder/upcoming/take_the_bait.txt (hunk 1 above)
Comment (Morgenmvffel) [reply forge-gui/res/cardsfolder/upcoming/take_the_bait.txt]: added

**Review by tool4ever — APPROVED** [review]
looks good, thanks!

**Pull request merged**